src/interfaces/jira.ts

```diff
@@ -8,6 +10,13 @@ interface JiraPullRequestRef {
 	uri: string;
 }
 
+export interface BulkSubmitRepositoryInfo {
```

Comment:
3 payloads have this: bulk submit commits, branches and PRs. This tiny refactoring allowed me to create `transform-repository`